hibernate4/src/test/java/com/fasterxml/jackson/datatype/hibernate4/HibernateTest.java

```
@@ -90,6 +91,7 @@ public void testCyclesJson() throws Exception {
         
         // false -> no forcing of lazy loading
         ObjectMapper mapper = mapperWithModule(false);
+        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

But wouldn't this hide possible errors? It should not be necessary to do this.